src/rendering/renderengine.cpp

```diff
@@ -1456,18 +1475,23 @@ void RenderEngine::renderScreenLog() {
                     return glm::vec4(0.3f, 0.3f, 0.85f, alpha);
                 default:
                     return white;
-            }
-        }(e->level);
-
-        RenderFont(
-            *_fontLog,
-            glm::vec2(
-                10 + 30 * _fontLog->pointSize(),
-                _fontLog->pointSize() * nr * 2 + fontRes.y * _verticalLogOffset
-            ),
-            fmt::format("({})", lvl),
-            color
-        );
+                }
+            }(it->level);
```

Comment:
Is this lambda here for performance reasons?

Reply:
I don't think it has any impact on the performance, but it allows us to make the `color` variable `const` and ensures that it is initialized.  IIFE ("iffy") is one of those JavaScript best-practices that i really like:
https://en.wikipedia.org/wiki/Immediately_invoked_function_expression